docs/components/movement-sensor/adxl345.md

```diff
@@ -129,7 +128,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
       "namespace": "rdk",
       "attributes": {
         "board": "local",
-        "i2c_bus": "default_i2c_bus",
+        "i2c_bus": "1",
```

Comment:
This says "1", but line 60 gave the example "2" and the new screenshot uses "2". Should they be different to show that you can have different values, or the same to be consistent? Maybe it's right the way you've written it; I'm just looking for weird stuff.

Reply:
I'm good with changing it to 1, just didn't catch that difference!